src/Command/GuideCommand.php

```diff
@@ -172,7 +169,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $out = $input->getOption('output');
         if (!$out) {
-            $style->block($content);
+            $out->write($content);
```

Comment:
```suggestion
            $style->write($content);
```
Or if you prefer using `$output`, the `$style` variable can be removed and `$stderr` can be replaced by `$stderr = new SymfonyStyle($input, $output->getErrorOutput())`.

Reply:
I don't want stderr here.

Reply:
So use `$style` or `$output`, not `$out` cause it's the `--output` option